webapp/src/containers/WalletPage/service.tsx

```diff
@@ -69,6 +76,39 @@ export const handelRemoveReceiveTxns = (id, networkName) => {
   return paymentData;
 };
 
+export const handleAddToken = (tokenData) => {
+  const networkType = getNetworkType();
+  const initialData = JSON.parse(PersistentStore.get('tokenInfo') || '{}');
+  const keyData = [...(initialData[networkType] || []), tokenData];
+  initialData[networkType] = keyData;
+  PersistentStore.set('tokenInfo', initialData);
+  return initialData[networkType];
+};
+
+export const getWalletToken = () => {
+  const networkType = getNetworkType();
```

Comment:
I think we can create a common method for these two lines. It's been used for the other methods as well.